core/ui/qr/upload/UploadQrPage/UploadQrPageWithoutVault.tsx

```
@@ -7,15 +7,15 @@ import { StyledPageContent } from '@lib/ui/qr/upload/UploadQRPage/UploadQRPage.s
 import { useCallback, useState } from 'react'
 import { useTranslation } from 'react-i18next'
 
-import { useAppNavigate } from '../../../../navigation/hooks/useAppNavigate'
+import { useCoreNavigate } from '../../../navigation/hooks/useCoreNavigate'
 import { UploadQrView } from '../UploadQrView'
 
 const uploadQrViews = ['scan', 'upload'] as const
 type UploadQrView = (typeof uploadQrViews)[number]
```

Biome (1.9.4)

[error] Shouldn't redeclare 'UploadQrView'. Consider to delete it or rename it.

'UploadQrView' is defined here:


(lint/suspicious/noRedeclare)